data/ecosystems/t/ton.toml

```diff
@@ -822,35 +922,62 @@ url = "https://github.com/DavidLee9291/ton-testnet.github.io"
 [[repo]]
 url = "https://github.com/DavidLee9291/tonHack"
 
+[[repo]]
+url = "https://github.com/dedust-io/DefiLlama-Adapters"
+
+[[repo]]
+url = "https://github.com/dedust-io/defillama-server"
```

Comment:
this doesn't look like TON? let's remove it maybe

Reply:
Done